Justfile

```diff
@@ -96,7 +100,7 @@ check:
 
 test-unit-mina-rs:
   @echo "--- Performing long-running mina-rs unit tests"
-  cd rust && time cargo nextest run --features mina_rs
+  cd rust && time cargo nextest run --release --features mina_rs
```

Comment:
Why??

Reply:
because we do the release build for tier3 so we might as well use that binary for the unit tests